random-beans/src/main/java/io/github/benas/randombeans/randomizers/collection/MapRandomizer.java

```diff
@@ -49,16 +49,6 @@
     private final Randomizer<V> valueRandomizer;
 
     /**
-     * Create a new {@link MapRandomizer} with a random number of entries.
-     *
-     * @param keyRandomizer   the randomizer for keys
-     * @param valueRandomizer the randomizer for values
-     */
-    public MapRandomizer(final Randomizer<K> keyRandomizer, final Randomizer<V> valueRandomizer) {
```

Comment:
These randomizer constructors and static factory methods are part of the API random beans exposes to it's users. To remove them means to break user code. This should not be done without a good reason and even then should only be done for the next major version (4.0).